challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -44,6 +58,18 @@ export default {
   data: () => ({
     engagementCenterEnabled: eXo.env.portal.engagementCenterEnabled,
     tab: null,
+    isAdministrator: false,
+    loading: true,
   }),
+  created() {
+    this.init();
+  },
+  methods: {
+    init() {
+      this.$realizationsServices.getAllRealizationsStatus()
+        .then(resp => this.isAdministrator = resp);
+      this.loading = false;
```

Comment:
You stopped loading just after sending the `fetch` operation which is an asynchronous operation. Which means, even if the fetch operation takes 10 seconds, the loading effect will be displayed 1 or 2 millisecond only because `this.loading` has become `false` just after launching the fetch asynchronous operation.

Besides, you may not need to use loading effect of the Data Table itself (which is better), but it would be better to use the general loading effect triggered and managed as shown in the following suggestion : 

```suggestion
      document.dispatchEvent(new CustomEvent('displayTopBarLoading'));
      this.$realizationsServices.getAllRealizationsStatus()
        .then(resp => this.isAdministrator = resp)
        .finally(() => document.dispatchEvent(new CustomEvent('hideTopBarLoading')));
```

Reply:
thank you for your explanation, extremely fruitful for me !